All Core Devs Meetings/Meeting 118.md

```diff
@@ -0,0 +1,546 @@
+
+**Tim**
+* Okay, I don't have a strong opinion, so if people prefer not to keep Shanghai, I guess. William, there's one opposition to keep in Shanghai. Referring to the Future Fork, but 
+
+**Marius**
+* Yeah, I would also oppose. I would also say that it's cooler to have the three different names and merge them if needed and to keep Shanghai as the feature fork regardless of whether it happens in December or q1 2022. 
+
+**Martin**
+* Yeah, I will post the opposition. I also lean towards having whatever, whatever is in the next fork and we don't know what's in it, but it's Shanghai. Makes it, I think it makes it easier. 
+
+**Tim**
+* Okay, I guess the seems pretty split do. It's like a weird way to phrase the questions, but do people kind of is it worth bringing this to say like the infrastructure called her and all core devs. And actually talking more with people who proposed eips than with core devs and kind of see what they prefer because I guess it feels like in a way, you know, we will kind of figure it out no matter what it is, and it's kind of our job to stay on top of this thing. But maybe kind of the perspective of People who are actually submitting stuff into these is you know, what makes it easier for them so this. 
+
+**Micah Zoltu**
+* Yeah. I think it also has an impact on just the broader Community people's people. Often talk about London, for example, long before London comes out and So, I think, I think they also are stakeholders that matter, like it shows up in news, article shows up in chat rooms, it shows up and Twitter tweets, like these names show up everywhere and they affect a lot of people. So yeah. 
+
+**Tim**
+* So with that being said, you think the time like Shanghai in the, in December, is it's just easier for people to keep track of 
+
+**Micah Zoltu**
+* That's my feeling but like if we wanted to solicit external opinions, I think that we should extend beyond just EIP authors and also trying to see if we can get feelings from the broader Community which I know is extremely hard. 
+
+**Martin**
+* So, the didn't we didn't kind of decide to use the dev com sites as the name for hard works just so we wouldn't have to be by shutting about names, so much in the future. I'm crazy. Why we should deviate from that on that? 
+
+**Pooja | ECH**
+* That was the first time when we decided that we will be following that Devcon names. And right after Istanbul, we did Muir Glacier because that was just the one Fork to push the difficulty Bob and I feel that it makes sense. If it is only for difficulty bomb, we should have something like that was proposed, like, our Arctic Glacier just to be in sync that. Okay, fine. It is just for difficulty bomb. And for all other feature folks that are very important for people to upgrade their nodes especially like with the latest of client it makes sense. I mean that's just my thought and it's not like very, very hard 
```

Comment:
```suggestion
* Istanbul was the first time when we decided that we will be following that Devcon names. And right after Istanbul, we did Muir Glacier because that was just one proposal to push the difficulty bomb and I feel that it makes sense. If it is only for difficulty bomb, we should have something like that was proposed, like, our Arctic Glacier just to be in sync that okay, fine. It is just for the difficulty bomb. And for all other feature forks that are very important for people to upgrade their nodes especially like with the latest clients, it makes sense. I mean that's just my thought and it's not like very, very strong suggestion.
```